<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>420</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDeviceManager" >
      <attribute name="title" >
       <string>Devices</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Connected devices. Doubleclick to change default remote</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="viewDeviceList" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExecution" >
      <attribute name="title" >
       <string>Execution Engine</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Application profiles loaded by the execution engine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="viewExecution" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite" >
      <attribute name="title" >
       <string>Running targets</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;table border="0" style="-qt-table-type: root; margin-top:3px; margin-bottom:3px; margin-left:4px; margin-right:4px;">
&lt;tr>
&lt;td style="border: none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Applications on the bus that you want to control&lt;/p>&lt;/td>&lt;/tr>&lt;/table>&lt;/body>&lt;/html></string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="viewTargetList" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Device manager:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="lblDevManState" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>If lirid is offline then it does not necessarly mean something bad.
You may just not have plugged in any supported device.
You can find a list of installed drivers in the drivers tab.</string>
       </property>
       <property name="styleSheet" >
        <string notr="true" >QLabel#lirid_state {color: rgb(255, 0, 0); }</string>
       </property>
       <property name="text" >
        <string>Undefined</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Dustismo Roman'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Last hal callout:&lt;br />This mechanism is important for starting the device manager&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">if it is not already running and you plug in a supported receiver.&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="text" >
        <string>Starter:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="lblDevManLastCallout" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;table style="-qt-table-type: root; margin-top:3px; margin-bottom:3px; margin-left:4px; margin-right:4px;">
&lt;tr>
&lt;td style="border: none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Hardware Abstraction Layer helper application that&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">starts the device manager if devices are plugged in.&lt;/p>&lt;/td>&lt;/tr>&lt;/table>&lt;/body>&lt;/html></string>
       </property>
       <property name="styleSheet" >
        <string notr="true" >QLabel#lirid_state {color: rgb(255, 0, 0); }</string>
       </property>
       <property name="text" >
        <string>Undefined</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Execution engine:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="lblExecutionState" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;table border="0" style="-qt-table-type: root; margin-top:3px; margin-bottom:3px; margin-left:4px; margin-right:4px;">
&lt;tr>
&lt;td style="border: none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Running in the context of this user. Executes commands on key presses.&lt;/p>&lt;/td>&lt;/tr>&lt;/table>&lt;/body>&lt;/html></string>
       </property>
       <property name="styleSheet" >
        <string notr="true" >QLabel#lirid_state {color: rgb(255, 0, 0); }</string>
       </property>
       <property name="text" >
        <string>Undefined</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
